doc/nrf/protocols/lte/index.rst

```diff
@@ -1,230 +1,17 @@
 .. _ug_lte:
 
-LTE
-###
+Cellular
+########
 
-.. contents::
-   :local:
-   :depth: 2
+Cellular technology enables devices to be connected to a base station without an intermediate device or gateway.
+The latest evolution of cellular networks is the fifth-generation (5G) wireless broadband standard built on existing technologies.
```

Comment:
Our cellular products are 4G, so maybe it would be better to talk about LTE and 4G here.

Reply:
I suggest we mention both as currently it is in 4G, but will be adopted in 5G

Reply:
Discussed with Jani M., keeping "5G" here is fine for him, so I guess no changes are then needed.